### PR TITLE
[Doc] update data cache, classic mode word usage, and unit

### PR DESCRIPTION
**Pull request by evelynzhaojie**
Signed-off-by: evelynzhaojie <everlyn.[zhaojie@gmail.com](mailto:zhaojie@starrocks.com)>

1. update data cache according to pr from rd https://github.com/StarRocks/docs.zh-cn/pull/6557/files
2. delete "classic mode" for word consistency
3. modify unit "pcs"

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/administration/Monitor_and_Alert.md [hunk 1]
```diff
@@ -8,75 +8,75 @@ The available metrics are:
 
 |Indicator|Unit|Type|Meaning|
 |---|:---:|:---:|---|
-|be_broker_count|pcs|average|Number of brokers |
-|be_brpc_endpoint_count|pcs|average|Number of StubCache in BRPC|
+|be_broker_count|count|average|Number of brokers |
```
Comment (kevincai) [inline docs/administration/Monitor_and_Alert.md:11]: can we unify the format, whether need to capitalize the first letter? I saw some are bytes/s, some are Byte, some are Ms.

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/administration/Monitor_and_Alert.md (hunk 1 above)
Comment (evelynzhaojie) [reply docs/administration/Monitor_and_Alert.md:11]: use the lower case for now. and we will update the metrics in these two months

Comment (evelynzhaojie) [reply docs/administration/Monitor_and_Alert.md:11]: use the lower case for now. and we will update the metrics in these two months

> can we unify the format, whether need to capitalize the first letter? I saw some are bytes/s, some are Byte, some are Ms.

use the lower case for now. and we will update the metrics in these two months

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.2

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**